GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -169,6 +169,10 @@ @implementation GIDSignIn {
 
 #pragma mark - Public methods
 
+// Handles the custom URL returned from AppAuth or Device Policy App (EMM flow).
```

Comment:
I think also it would be more clear to use a different word than 'handle' in the description since that is the word used in the method.

Reply:
Replaced with `open`.